addons/mail/static/tests/discuss/message_pin/pinned_messages_tests.js

```diff
@@ -138,11 +134,12 @@ QUnit.test("Jump to message from notification", async (assert) => {
         });
     }
     const { openDiscuss } = await start();
-    await openDiscuss(channelId);
-    await click(".o-mail-Message:eq(0) [title='Expand']");
-    await click(".dropdown-item:contains(Pin)");
-    await click(".modal-footer button:contains(pin it)");
-    await click(".o_mail_notification a:contains(message)");
-    await nextTick();
-    assert.isVisible($(".o-mail-Message:contains(Hello world!)"));
+    openDiscuss(channelId);
+    await clickContains(":nth-child(1 of .o-mail-Message) [title='Expand']");
+    await clickContains(".dropdown-item", { text: "Pin" });
+    await clickContains(".modal-footer button", { text: "Yeah, pin it!" });
+    await contains(".o_mail_notification");
```

Comment:
Just asserting presence of `.o_mail_notification` is functionally not meaningful enough. Would be best to show the text of the notification, to see a clearer reason to assert presence of such notification

Reply:
I just backported the test from master (btw the test was skipped because it did not pass^^). But I will change this as well :)